app/views/layouts/application.html.erb

```diff
@@ -39,8 +39,9 @@
           </ul>
           <ul class="nav navbar-nav navbar-right">
             <% if current_user %>
-              <li class="navbar-brand">Logged in as <%= current_user.username %></li>
-              <li><%= link_to "Logout", logout_path %></li>
+              <li class="logged-as-navbar"><%= "Logged as" %></li>
+              <li class="username-navbar"><%= link_to " #{current_user.username} |"%></li>
```

Comment:
It seems like this would need to be `<li class="username-navbar"><%= link_to " #{current_user.username} |", dashboard_path %></li>`